<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>yahtzeeSolver</class>
 <widget class="QMainWindow" name="yahtzeeSolver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>yahtzeeSolver</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="rollLabel">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>150</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upDice">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="exLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="downDice">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lockIn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>lockIn</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Dice">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
